extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -78,6 +84,8 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 			slotName = slot.name,
 			uapId = uapContext.getUapId();
 
+		console.log('pushAd', slot.name, uapId);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)